packages/apps/sendgrid/tsconfig.json

```
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "declaration": true
+  }
```

```suggestion
  "extends": "../../../tsconfig.json"
```

Ninguém deve executar funções desse pacote, não precisa de type declarations...